src/gui/Application.cpp

```diff
@@ -338,13 +363,15 @@ bool Application::sendFileNamesToRunningInstance(const QStringList& fileNames)
     QByteArray data;
     QDataStream out(&data, QIODevice::WriteOnly);
     out.setVersion(QDataStream::Qt_5_0);
-    out << quint32(0) << fileNames;
+    out << quint32(0); //reserve space for block size
+    out << id; //send command ID: 1 for file open, 2 for db lock
+    if (id == 1) out << fileNames; //send file names to be opened, if that is the command. TODO: make this not suck
```

Comment:
This will be deleted when you split the function, but all if statements need to use brackets.
```
if (id == 1) {
    ...
}
```